src/assets/style.css

```diff
@@ -0,0 +1,381 @@
+}
+
+section#works > .cards > .specialCard > button {
+  background-color: #ff6b00;
+  width: fit-content;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  padding: 12px;
+  border: 1px solid transparent;
+}
+
+section#works > .cards > .card > button {
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  padding: 12px 0;
+  border: 1px solid transparent;
+  width: 100%;
+  margin-top: 10px;
+}
+
+section#about > .info > button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  cursor: pointer;
+}
+
+section#about > .info > button:active {
+  background: #e05304;
+  border: 1px solid transparent;
+  color: #f4f1f0;
+  cursor: pointer;
+}
+
+section#works > .cards > .specialCard > button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  cursor: pointer;
+}
+
+section#works > .cards > .specialCard > button:active {
+  background: #e05304;
+  border: 1px solid transparent;
+  color: #f4f1f0;
+  cursor: pointer;
+}
+
+section#works > .cards > .card {
+  background-image: url("./worksBg.svg");
+}
+
+section#works > .cards > .card > .information {
+  padding: 0 16px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 12px;
+}
+
+section#about > .hardSkills > ul > li:first-child > h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+section#works > .cards > .card > .information > h3 {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-top: 84px;
+}
+
+section#works > .cards > .card > .information > p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+section#works > .cards > .card > .information > ul.languages {
+  display: flex;
+  list-style-type: none;
+  column-gap: 12px;
+}
+
+section#works > .cards > .card > .information > ul.languages > li {
+  padding: 10px 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+section#works > .cards > .card > button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  cursor: pointer;
+}
+
+section#works > .cards > .card > button:active {
+  background: #e05304;
+  border: 1px solid transparent;
+  color: #f4f1f0;
+  cursor: pointer;
+}
+
+/* WORKS SECTION ENDED */
+
+/* ABOUT SECTION STARTED */
+
+section#about {
+  padding-top: 120px;
+  padding-bottom: 120px;
+  background: #1c1a19;
+}
```

Comment:
- [ ] Please align the content inside this `section#about` selector with flexbox. You have instead aligned only the about-me div with flex.